src/components/templates/admin/news/form.tsx

```diff
@@ -21,50 +22,39 @@ export default function NewsFormTemplate({ news }: Props) {
     formState: { errors },
   } = useForm<News>();
 
-  const onSubmit: SubmitHandler<News> = (data) => {
-    const db = firebase.firestore();
+  const onSubmit: SubmitHandler<News> = async (data) => {
+    var newsID = "";
```

Comment:
=> let

Reply:
> var => let

Question: lúc em đọc tài liệu, thì em đã nghĩ là chỗ này dùng var hay let đều được, nhưng tại sao mình phải dùng let vậy chị?

Reply:
@duoctvd 
Thường thì chỉ được dùng let và const thôi, phạm vi của var nó lớn hơn let nên dễ sai sót hơn.
https://www.geeksforgeeks.org/difference-between-var-and-let-in-javascript/#:~:text=var%20and%20let%20are%20both,program%20as%20compared%20to%20let.